src/api/versions/v1/services/authentication-service.ts

```
@@ -223,4 +224,17 @@ export class AuthenticationService {
 
     return user;
   }
+
+  private async ensureUserNotBanned(user: UserKV): Promise<void> {
+    if (user.ban) {
+      const { expiresAt, reason } = user.ban;
+
+      if (expiresAt !== null && expiresAt < Date.now()) {
+        delete user.ban;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)